sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py

```diff
@@ -206,6 +206,8 @@ class CategorizedEntity(DictMixin):
     :vartype category: str
     :ivar subcategory: Entity subcategory, such as Age/Year/TimeRange etc
     :vartype subcategory: str
+    :param int offset: The entity text offset from the start of the sentence.
+    :param int length: The length of the entity text.
```

Comment:
should be ivar (there's a few instances on this file)

Reply:
yeah, sorry about that